tests/test_main.py

```diff
@@ -1 +1,88 @@
-# write your code here
+from app.main import get_human_age
+
+
+def test_output_type():
+    assert type(get_human_age(1, 1)) == list
+
+    is_int = True
+    if type(get_human_age(1, 1)) == list:
+        for i in get_human_age(30, 31):
+            if not type(i) == int:
+                is_int = False
+                break
+
+    assert is_int
```

Comment:
It can be more shorted with `assert all([isinstanse(value, int) for value in get_human_age(30, 31)])`